conftest.py

```diff
@@ -68,3 +69,24 @@ def example_container():
         container=container["Id"],
         force=True
     )
+
+
+def example_container_with_docker_compose():
+    subprocess.check_output(shlex.split("docker-compose up -d"))
+
+    docker_client = _docker_client()
+    container_info = docker_client.inspect_container("examplecontainer")
+    # Get the IP address using the network docker-compose automatically creates
+    yield container_info["NetworkSettings"]["Networks"]["pytestdockerpy_default"]["IPAddress"]
+
+    subprocess.check_output(shlex.split("docker-compose down"))
+
+
+@pytest.yield_fixture(params=[
+    "use_docker_py",
+    "use_docker_compose"])
```

Comment:
@mtomwing do you have any better ideas how I could do this? (i.e. use a fixture to get the test to run twice with the two distinct ways of running the tests, but without hacky hard coded flags)